### ISSUE TITLE
MGS MC V's Fix?

### ISSUE THREAD

**Issue by Jake0light** [VFansss/mgs2-v-s-fix#212]
**the** **title says it all**
can we please have it reworked for steam version or aka metal gear solid 2: son of liberty master collection
im just curious if it is possible at all :)

**Comment by VFansss** [comment]
I didn't tried it, but I'm pretty confident that THIS version doesn't work (I guess .exe are different and a lot of things works differently).

A LOT of community fixes already exist for MGS MC (Internal resolution, Deck support), but...there are several things that still need to be addresses: high fidelity audio and OFFICIAL Proton support, to say some.

Honestly, I want to wait: there's nothing worse than doing something and then found that's useless because they fixed officially.

Right now PC MGS MC would need a "V's Fix"? Uhm, certainly yes. But in 2 months?

I bet not.

Let's see what Konami will officially patch: I honestly believe that Konami will patch by themselfs most of these things (Valve is working to make it works on Proton on their end).

Also mind that I know how to do a lot of things, but I'm not a reverse engineer.

Let's see how Konami and community cope together and, if needed, I surely would like to release a MC fix, if needed.

This also mean that I should rework...well, A LOT of different things on the codebase and overall design: V's Fix is kinda bad coded and is kinda "Windows" focused: a LOT of things changed in all these years.

Let's say that if something is not fixed before Christmas, I'm pretty sure they will not be fixed officially.

Let's see...

**Comment by Jake0light** [comment]
fair enough i thought i ask since the aiming really tilts me sometimes so thanks for a reply i hope they officially do something but if not i hope its easy for you to do it

**Comment by ShizCalev** [comment]
Definitely an old issue, but feel free to open issue reports for any issues that need correcting in the MC version over on MGSHDFix's repo;
https://github.com/Lyall/MGSHDFix/issues

Likewise, there's a community bug tracker going here:
[Metal Gear Solid Master Collection - Community Bug Tracker](https://docs.google.com/spreadsheets/d/1WhQSRpkC_A9wBDV0o-Pohh1dMhL1H6nbVzvdluIVWrw/edit?gid=0#gid=0)
